_data/projects/keep.yml

```
@@ -0,0 +1,12 @@
+name: Keep
+desc: Open-source alert management and workflow automation platform.
+site: https://www.keephq.dev
+tags:
+  - monitoring
+  - alerting
+  - alerts
+  - workflow-automation
+  - automation
+upforgrabs:
+  name: good first issues
```

I think this is the problem - the `s` on the end is not actually part of the label

```suggestion
  name: good first issue
```